services/director-v2/src/simcore_service_director_v2/modules/director_v0.py

```diff
@@ -110,6 +116,17 @@ async def get_service_extras(self, service: ServiceKeyVersion) -> ServiceExtras:
             return ServiceExtras.parse_obj(unenvelope_or_raise_error(resp))
         raise HTTPException(status_code=resp.status_code, detail=resp.content)
 
+    @log_decorator(logger=logger)
+    async def get_service_settings(self, service: ServiceKeyVersion) -> ServiceSettings:
+        # TODO: DYNAMIC-SIDECAR: refactor to use endpoint defined by ANE
+        resp = await self.request(
+            "GET",
+            f"service_labels/{urllib.parse.quote_plus(service.key)}/{service.version}",
```

Comment:
there is something weird here. too many functions for the same thigns? 
- service_labels/key/version
- services/key/version:labels

one of them is too much. the first one. and correct the second one to use a / instead of a :

Reply:
this wan I've removed it it was not used